### ISSUE TITLE
[IQKeyboardManager init] takes a long time in App Launch

### ISSUE THREAD

**Issue by sheep-q** [hackiftekhar/IQKeyboardManager#2066]
**Describe the bug**
When I Profile my app via Instrument and choose the App Launch template, after recording, I see problems in phase "Launching - UIKit Initialization"
[IQKeyboardManager init] took a long time in App Launch, about 43.05 ms, which equals 48.2% of the total time. 
I used many OSS frameworks and libraries and as I saw, just func init of IQKeyboardManager impacted this phase. it made the app launch run longer than expected. 
<img width="1258" alt="Screenshot 2024-08-12 at 11 23 09" src="https://github.com/user-attachments/assets/e8797bd6-5107-4c89-8185-8a8d5512013b">

**To Reproduce**
1. Profile app with release mode 
2. Chose App Launch template 
3. Record the app till finishing launching.
4. See metrics in the App Life Cycle information.

**Expected behavior**
I expect that reducing time to run func init, or func init no longer impact this phase too (like other OSS libraries, because I didn't see any impacts of other OSS libraries in this phase)

**Screenshots**
<img width="1258" alt="Screenshot 2024-08-12 at 11 23 09" src="https://github.com/user-attachments/assets/77ba64dd-f630-4da8-b8fe-a38caac500af">

**Demo Project**
If applicable, add demo project.

### Versions
**Xcode:** 15.2
**Mac OS:** Sonoma 14.2
**Simulator/Device:** iPhone 8+ 16.7.8
**Library Version:** IQKeyboardManager (6.5.6)

**Additional context**

**Comment by quangfnv-1159** [comment]
Filter "Hide System Libraries" to see the effects of OSS libraries.
<img width="701" alt="Screenshot 2024-08-12 at 11 44 34" src="https://github.com/user-attachments/assets/37216981-bced-4434-8b62-007b8438a693">

**Comment by hackiftekhar** [comment]
This is a known issue because the library has to subscribe for keyboard, textfield, didBecomeActive. It has to do first time setup, this is why it takes time.

You should be able to reduce this delay by moving this to a dispatchQueue something like below which will run after your current operations:-

```objc
    dispatch_async(dispatch_get_main_queue(), ^{
        [[IQKeyboardManager sharedManager] setEnable:YES];
    });
```

Or you should move `setEnable` function call from AppDelegate to somewhere else when your screen starts to show.

```objc
 [[IQKeyboardManager sharedManager] setEnable:YES];
```

**Comment by quangfnv-1159** [comment]
Let me provide more details. 
The problem happened in phase "Launching - UIKit Initialization"-  which is even executed before func didFinishLaunchingWithOptions() in AppDelegate is called. 
I saw the library implemented `+load `method. And the `+load` method in Objective-C is a special method that is called automatically by the Objective-C runtime when a class is initially loaded into memory. This method is called before the application’s main function is executed, which means it can affect the app launch time.

According to your suggestion, I implemented it and saw it doesn't change the result.  So I think to reduce launching time, `init` func should be lazy or async loaded.

I updated the library to the newest compatible version for Swift 4.2 - [6.5.16](https://github.com/hackiftekhar/IQKeyboardManager/releases/tag/6.5.16), the app launch time seems to be faster. The difference from the old version is that UIKit-related functions are run async instead of sync (addDoneOnKeyboardWithTarget & addPreviousNextDoneOnKeyboardWithTarget). It's doesn't affect to phase "Launching - UIKit Initialization" anymore.

The problem was solved in version [6.5.16](https://github.com/hackiftekhar/IQKeyboardManager/releases/tag/6.5.16) to me. Thank you for your hard working.
